e2e/cli-e2e/tests/__snapshots__/collect.e2e.test.ts.snap

```diff
@@ -1696,3 +1696,2021 @@ exports[`CLI collect > should run ESLint plugin and create report.json 1`] = `
+          "score": 1,
+          "slug": "is-on-https",
+          "title": "Uses HTTPS",
+          "value": 0,
+        },
+        {
+          "description": "A \`<meta name="viewport">\` not only optimizes your app for mobile screen sizes, but also prevents [a 300 millisecond delay to user input](https://developer.chrome.com/blog/300ms-tap-delay-gone-away/). [Learn more about using the viewport meta tag](https://developer.chrome.com/docs/lighthouse/pwa/viewport/).",
+          "score": 1,
+          "slug": "viewport",
+          "title": "Has a \`<meta name="viewport">\` tag with \`width\` or \`initial-scale\`",
+          "value": 0,
+        },
+        {
+          "description": "First Contentful Paint marks the time at which the first text or image is painted. [Learn more about the First Contentful Paint metric](https://developer.chrome.com/docs/lighthouse/performance/first-contentful-paint/).",
+          "displayValue": "0.8 s",
+          "score": 1,
+          "slug": "first-contentful-paint",
+          "title": "First Contentful Paint",
+          "value": 830,
+        },
+        {
+          "description": "Largest Contentful Paint marks the time at which the largest text or image is painted. [Learn more about the Largest Contentful Paint metric](https://developer.chrome.com/docs/lighthouse/performance/lighthouse-largest-contentful-paint/)",
+          "displayValue": "2.0 s",
+          "score": 0.97,
+          "slug": "largest-contentful-paint",
+          "title": "Largest Contentful Paint",
+          "value": 1989,
+        },
+        {
+          "description": "First Meaningful Paint measures when the primary content of a page is visible. [Learn more about the First Meaningful Paint metric](https://developer.chrome.com/docs/lighthouse/performance/first-meaningful-paint/).",
+          "displayValue": "0.8 s",
+          "score": 1,
+          "slug": "first-meaningful-paint",
+          "title": "First Meaningful Paint",
+          "value": 842,
+        },
+        {
+          "description": "Speed Index shows how quickly the contents of a page are visibly populated. [Learn more about the Speed Index metric](https://developer.chrome.com/docs/lighthouse/performance/speed-index/).",
+          "displayValue": "1.0 s",
+          "score": 1,
+          "slug": "speed-index",
+          "title": "Speed Index",
+          "value": 1038,
+        },
```

Comment:
I don't think we can include the `score`, `value` and `displayValue` in the Lighthouse snapshot, it varies too much.

![image](https://github.com/code-pushup/cli/assets/34691111/f6b94aef-ff8e-46c5-897a-04aa35b15d8b)

Reply:
added a helper